wagtail/admin/static_src/wagtailadmin/scss/_variables.scss

```diff
@@ -25,9 +25,10 @@ $breakpoint-desktop-large: 75em; // 1200px
 $breakpoint-desktop-larger: 100em; // 1600px
 
 // colours
-$color-teal: #43b1b0;
-$color-teal-darker: darken($color-teal, 10%);
-$color-teal-dark: #246060;
+$color-teal: #007d7e;
+$color-teal-darker: darken(adjust-hue($color-teal, 1), 3);
```

Comment:
I'm very much in favor of the change to using color functions! My personal preference would be to add a code comment reminding that if the base colors change in the future the functions may also need adjusting, but that's far from necessary.